dns-nas-utils/etc/dns-nas-utils.conf

```diff
@@ -0,0 +1,18 @@
+#Use uncommented lines to change daemon configuration.
+
+# High frequency sleep time (low frequency is high*10)
+#frequency=0.75
+
+# Fan hysteresis, low and high thresholds.
+#low_thresh=45
+#high_thresh=50
+#hysteresis=2
+
+# Temporary path for some file exchanges between different calls.
+#tmppath="/run/dns-nas-utils"
+
+# Default fan device
+#fan_device="/sys/devices/gpio_fan/hwmon/hwmon0/fan1_target"
```

Comment:
@dobiatowski What's the path? Adding various paths to search for is probably a good idea. Or it could be the default is just wrong nowadays.